tests/scripts/thread-cert/node.py

```diff
@@ -989,8 +989,8 @@ def srp_client_get_host_name(self):
         self.send_command(f'srp client host name')
         self._expect_done()
 
-    def srp_client_remove_host(self, remove_key=False):
-        self.send_command(f'srp client host remove {"1" if remove_key else "0"}')
+    def srp_client_remove_host(self, remove_key=False, send_unreg_to_server=False):
```

Comment:
```suggestion
    def srp_client_remove_host(self, remove_key:bool=False, send_unreg_to_server:bool=False):
```

The error could have been avoided if typing were used.

Reply:
It's a good idea. But I don't see other functions/methods in `node.py` to use the types arg model. Would be weird to just change this one.

I think there is room for clean up in the `node.py` module (immediate things I see: some method had docs, some don't, the naming style is not consistent)... I think in a future PR perhaps (when/if someone has some cycles to add them 😃).